src/models/sensor-definitions.ts

```diff
@@ -64,7 +64,7 @@ export const SensorStrings:ISensorStrings = {
       "data_collection_complete": "Data collection complete.",
       "disconnected": "Disconnected.",
       "unexpected_error": "There was an unexpected error when connecting to the sensor.",
-      "sensor_connector_not_responding": "The SensorConnector application is not responding. To continue data collection click the \"Launch SensorConnector\" button at the bottom of the dialog or later click the corresponding button at the bottom of the window.",
+      "sensor_connector_not_responding": "The SensorConnector application is not responding. To continue data collection click the \"Launch SensorConnector\" button at the bottom of the dialog.",
```

Comment:
You can probably remove "at the bottom of the dialog" as well, since there's no longer any ambiguity about which one we're referring to.

Reply:
👍